lib/matplotlib/backend_managers.py

```diff
@@ -79,14 +82,15 @@ def __init__(self, figure, num):
         self.window = self._backend.Window('Figure %d' % num)
         self.window.mpl_connect('window_destroy_event', self.destroy)
 
-        self.canvas = self._backend.FigureCanvas(figure, manager=self)
+        self._figure = None
```

Comment:
Is not that we want to have no figure (when the figure will be changeable we'll want that option).
In general, I like to declare all the attributes at `__init__` and I am not doing `self._figure = self._set_figure(figure)` because I want to transform that latter into a property setter.